### DISCUSSION TITLE
Bot Composer handover to MS teams

### DISCUSSION THREAD

**Discussion by aparnaji** [microsoft/BotFramework-Composer#8083] [General]
Hi Where Can I find documentation/ information for BoT Composer handover to microsoft teams. ( user should be able to converse via Bot and Agent via  Microsoft Teams)

**Comment by tushaar9027** [comment]
As far as i know right now composer only supports two vendors  for now which have package available in .net version
1. Service Now Agent Chat
2. Live Person

Documentation Link:-
https://docs.microsoft.com/en-us/composer/concept-agent-handoff
